README.md

```diff
@@ -1,30 +1,91 @@
-# sn-treemap
-Treemap Supernova
-
-## Mobile
-### Preamble
-Mobile uses react-native.   As such all javascript should work on both the web and mobile. However react-native uses a custom JavaScript engine called Hermes.  For rendering picasso charts, mobile uses a hightly optimized rendering engine, called [Helium](https://github.com/qlik-oss/react-native-helium), that uses a GPU backend for Skia.  Skia is the same 2D graphics library used for Canvas in Chrome and Android.  So again, things must be tested to ensure compatiblity.
-
-### Testing mobile
-#### Pre-reqs
-1. For now a Mac System, until we have automated tests in the cloud
-2. Install [react-native dev environment](https://reactnative.dev/docs/0.68/environment-setup). This includes Xcode and Android Studio.
-3. [A QCS Account with an API Key](https://help.qlik.com/en-US/cloud-services/Subsystems/Hub/Content/Sense_Hub/Admin/mc-generate-api-keys.htm)
-4. An app with some treemaps
-
-
-#### Running
-** Please Note this can take a while to build if it's the first time **
-1. `yarn prepare-mobile`
-2. `yarn ios`
-
-## Nebula dev
-#### NOTE: Create an app using random data and create one treemap.
-1. clone me
-2. checkout master
-3. yarn
-4. yarn build:watch
-5. yarn start
+# @nebula.js/sn-treemap
+
+Treemaps display hierarchical data by using nested rectangles, that is, smaller rectangles within a larger rectangle. Use a treemap when space is constrained and you have a large amount of hierarchical data that you need to get an overview of. Treemaps should primarily be used with values that can be aggregated.
+
+## Requirements
+
+Requires `@nebula.js/stardust` version `1.2.0` or later.
```

Comment:
have you tested that stardust version (and maybe others)?
when we have the correct required version it should also be updated in peerDependencies in package.json